admin/src/components/SelectedCapabilities.jsx

```diff
@@ -0,0 +1,22 @@
+export default function SelectedCapabilities(props) {
+  const { selecteditems, toggleSelectItem } = props
+  return (
+    <div className="urlslab-selected-capabilities">
+      <h5 style={{ marginBottom: '15px' }}>selected</h5>
+      {selecteditems.map((item) => (
+        <div className="urlslab-capability-item">
+          <label role="Controlled Checkbox" className={`urlslab-checkbox `}>
```

Comment:
Simple class names doesn't need to be wrapped in brackets.
Use simple quotes `""` if not used variables inside string.